tools/osdfgen/osdfgen.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2018 the Service Broker Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/csv"
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"gopkg.in/src-d/go-license-detector.v2/licensedb"
+	"gopkg.in/src-d/go-license-detector.v2/licensedb/filer"
+
+	toml "github.com/pelletier/go-toml"
+)
+
+var (
+	out  = flag.String("o", "-", "Sets the output location of the OSDF csv")
+	proj = flag.String("p", ".", "the project root")
+)
+
+type Lockfile struct {
+	Projects []Project
+}
+
+type Project struct {
+	Name     string
+	Revision string
+}
+
+func main() {
```

Comment:
Consider adding a doc header explaining the usage either here or as a package header.